aws-api/src/androidTest/java/com/amplifyframework/api/aws/GraphQLLazySubscribeInstrumentationTest.kt

```diff
@@ -44,24 +45,44 @@ import org.junit.Test
 class GraphQLLazySubscribeInstrumentationTest {
 
     companion object {
+
+        lateinit var instrumentationRunId: String
```

Comment:
Suggestion:  Can we initialize this property using 'by lazy' delegate?
Something like:  val instrumentationRunId: String by lazy {
        UUID.randomUUID().toString()
    }

Reply:
i wanted to say the exact same thing.